assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -213,6 +198,37 @@ const Cart = ( {
 	const totalsCurrency = getCurrencyFromPriceResponse( cartTotals );
 	const totalRowsConfig = getTotalRowsConfig();
 
+	const ShippingCalculatorOptions = () => (
```

Comment:
I'm speaking about my personal taste, but I'm not a big fan of creating components inside other components' render function and I think it might have a performance impact if we do it regularly. I would rather prefer making it a const assignment `const shippingCalculatorOptions = (<fieldset>...</fieldset>);` or keep it inside the main return as it was.

If we still want to move it to a different component, I think it would be better to move it to its own file and pass it the necessary props.